modules/nf-core/agat/spmergeannotations/tests/main.nf.test

```diff
@@ -0,0 +1,71 @@
+nextflow_process {
+
+    name "Test Process AGAT_SPMERGEANNOTATIONS"
+    script "../main.nf"
+    process "AGAT_SPMERGEANNOTATIONS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "agat"
+    tag "agat/spmergeannotations"
+
+    test("candidatus_portiera_aleyrodidarum-multi_gffs") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    [
+                        file(params.test_data['candidatus_portiera_aleyrodidarum']['genome']['test1_gff'], checkIfExists: true),
+                        file(params.test_data['candidatus_portiera_aleyrodidarum']['genome']['test2_gff'], checkIfExists: true),
+                        file(params.test_data['candidatus_portiera_aleyrodidarum']['genome']['test3_gff'], checkIfExists: true),
+                    ]
+                ]
+
+                input[1] = []
```

Comment:
Can you make a test where a config is passed? It can be extremely minimal, just to see that this input works

Reply:
Progress is underway at: https://github.com/nf-core/modules/pull/5771

Reply:
@itrujnara 

This has been done now. Thank you!